docs/guide/ai-extension-install.md

```
@@ -26,7 +26,7 @@ AI Extension 是一个基于 WXT 框架开发的浏览器扩展插件，它提
 
 访问扩展下载地址，下载扩展 ZIP 文件：
 
-**下载地址**：https://ai.opentiny.design/extension.zip
+**下载地址**：https://docs.opentiny.design/download/extension.zip
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)